source/07-processing/03-counting-unique-items.rst

```diff
@@ -11,6 +11,14 @@ Some COUNTER Metric_Types count the number of unique items that had a certain ac
 
 For the purpose of COUNTER metrics, an item is the typical unit of content being accessed by users, such as articles, book chapters, book segments, whole books (if delivered as a single file), and multimedia content. The item MUST be identified using the unique ID which identifies the work (e.g. chapter or article) regardless of format (e.g. PDF, HTML, or EPUB). If no item-level identifier is available, then use the item name in combination with the identifier of the parent item (i.e. the article title + ISSN of the journal, or chapter name + ISBN of the book).
 
+The method for counting book usage in R5.1 at the item level is different than it was in R5. In R5.1, a Unique_Item_Investigation or Unique_Item_Request MUST be counted for each item (Book_Segment) that is used, where book content is available as Book_Segments.
+
+* Where books are available as separate Book_Segments, a Unique_Item_Request MUST be counted for each Book_Segment a user elects to download.
+* Where books are available as separate Book_Segments and as whole-book downloads, a Unique_Item_Request MUST be counted for each Book_Segment when a user elects to download the whole book as a single file.
+* Where books are only available as whole-book downloads (i.e. there are no Book_Segments), only 1 Unique_Item_Request MUST be counted.
```

Comment:
This rule should only apply when there is no information about the number of Book_Segments in the book available! Using an estimate or average isn't allowed, so this is the only option in that case. If the number of Book_Segments is known, it should be reported like in the previous case.

Reply:
Disagree - there is no option for users to select just one or a few book segments, the material is presented as all-or-nothing. In this instance the report providers should be using the book_segment extension.

Reply:
In my opinion that contradicts our R5.1 concept (at least how I understood it), that Book_Segment (or Reference_Item) is always the unit of reporting for books, independent from how the content is delivered. For the required reports TR and PR the content provider only needs the number of Book_Segments (no Book_Segment metadata) for reporting the usage according to this concept, and a content provider that has this information should report the usage in this way. If the content provider doesn't know the number of Book_Segments, that's of course a different case.

Reply:
Note that if only one Total/Unique_Item request would be counted if only whole book downloads are available, as the consequence that content provider would not be allowed to provide IR, even if the necessary Book_Segment metadata would be available, because that would result in an inconsistency between the reports.

Reply:
Sorry Bernd but this came through very clearly in the consultation responses from both report consumers and report providers: if the book is only available as a whole book, it should be reported as a single item. This was particularly of concern to aggregators who may have a mixture of metadata, knowing the book segment count for some titles but not all.

Reply:
[taken offline]